src/azure-cli/azure/cli/command_modules/identity/commands.py

```diff
@@ -31,3 +36,12 @@ def load_command_table(self, _):
 
     with self.command_group('identity', msi_operations_sdk, client_factory=_msi_operations_operations) as g:
         g.command('list-operations', 'list')
+
+    with self.command_group('identity federated-credential', federated_identity_credentials_sdk,
+                            client_factory=_msi_federated_identity_credentials_operations,
+                            min_api='2022-01-31-preview') as g:
+        g.custom_command('create', 'create_or_update_federated_credential')
+        g.custom_command('update', 'create_or_update_federated_credential')
+        g.custom_show_command('show', 'show_federated_credential')
+        g.custom_command('delete', 'delete_federated_credential')
```

Comment:
Do we need to consider adding `confirmation=True` for the delete command?

Reply:
Yes, adding a confirmation for `delete` command is better. I will add it.